source/EvoPlanet/EvoPlanet.Server/Controllers/SolarSystemController.cs

```diff
@@ -98,12 +106,13 @@ public async Task<IActionResult> GetAllSolarSystemsMongo()
             return Ok(solarSystems);
     }
 
+
+        //example: https://localhost:7081/api/SolarSystem/mongo/getOne667adf86b7477c4b4fc38de9
```

Comment:
Is this example right? No / (slash) after the getOne? Also, the [FromBody] suggests there is no id in the url.